src/library/scala/collection/IterableOnce.scala

```diff
@@ -1068,16 +1074,18 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
     *
     *  @example  `List(1, 2, 3).mkString("|") = "1|2|3"`
     */
-  def mkString(sep: String): String = mkString("", sep, "")
+  @inline final def mkString(sep: String): String = mkString("", sep, "")
 
   /** Displays all elements of this $coll in a string.
+    *
+    * Delegates to addString, which can be overriden.
     *
     *  @return a string representation of this $coll. In the resulting string
     *          the string representations (w.r.t. the method `toString`)
     *          of all elements of this $coll follow each other without any
     *          separator string.
```

Comment:
Worth scout's effort? `Concatenates the string representations (...) of all the elements of this $coll.`

Reply:
Also I'm not sure "Displays" is the right word. I would move the long return description to replace that first sentence, and then just `@return a string representation of this $coll`.

Reply:
you seem to be reviewing the entire file for errors, instead of my PR. Feel free to make those changes in a separate PR.

Reply:
You don't need to pursue scouting issues, but I'll raise them for the record. Folks seeing mkString for the first time, and I remember my first time, will say, "Man, I love this language."